app/assets/stylesheets/govuk_publishing_components/components/_option-select.scss

```diff
@@ -76,11 +76,9 @@
 
 .gem-c-option-select__icon {
```

Comment:
One thing that could happen when switching to use flex for the layout is that the icon will shrink depending on the length of the text, for example:
<img width="247" alt="Screenshot 2024-10-03 at 09 27 12" src="https://github.com/user-attachments/assets/a12d4b10-2a49-461c-a3a4-4fa4759c8dd3">

One possible option to stop the icon from shrinking is to add `flex: 0 0 30px`. I imagine the same CSS rule would then need to be added to the expander component in finder-frontend.

Reply:
Ah good spot, thanks! I've added a `flex-shrink: 0` to solve this rather than your suggestion, as it avoids the need to update the number if the size of the icon changes. @MartinJJones